packages/samples/headless-react/src/App.tsx

```diff
@@ -173,10 +194,35 @@ function App() {
           <FacetManager>
             <Facet field="author" facetId="author-1" />
             <Facet field="category" facetId="category-1" />
+            <NumericFacet
+              format={(bytes) => filesize(bytes, {base: 10})}
+              field="size"
+              facetId="size-1"
+              generateAutomaticRanges={true}
+            />
+            <NumericFacet
+              format={(bytes) => filesize(bytes, {base: 10})}
+              field="size"
+              facetId="size-2"
+              generateAutomaticRanges={false}
+              currentValues={[
+                buildNumericRange({start: 0, end: 5e3}),
+                buildNumericRange({start: 5e3, end: 50e3}),
+                buildNumericRange({start: 50e3, end: 5e6}),
```

Comment:
An alternative to this could be to create a `NumericRange` react component and make `NumericFacet` contain them.